org.civicrm.reqangular/src/common/templates/date-picker.html

```diff
@@ -0,0 +1,42 @@
+<div class="input-group chr_uib-date-picker">
+  <input
+    type="text"
+    class="form-control"
+    uib-datepicker-popup
+    ng-readonly="datePicker.ngReadonly"
+    ng-model="datePicker.ngModel"
+    ng-disabled="datePicker.ngDisabled"
+    ng-click="datePicker.isOpen = true"
+    ng-change="datePicker.ngModelChange()"
+    show-button-bar="datePicker.showButtonBar"
+    datepicker-options="datePicker.datepickerOptions"
+    is-open="datePicker.isOpen"/>
+  <span
+    class="input-group-addon pointer"
+    ng-disabled="datePicker.ngDisabled"
+    ng-click="datePicker.customNgClick ? datePicker.customNgClick : datePicker.isOpen = true">
+    <i class="fa fa-calendar"></i>
+  </span>
+</div>
+<div class="chr_html5-date-picker">
+  <input
+    required
+    class="form-control"
+    ng-disabled="datePicker.ngDisabled"
+    ng-change="datePicker.ngModelChange()"
+    ng-model="datePicker.ngModel"
+    min="{{datePicker.convertToHTML5DatepickerFormat(datePicker.datepickerOptions.minDate)}}"
+    max="{{datePicker.convertToHTML5DatepickerFormat(datePicker.datepickerOptions.maxDate)}}"
+    type="date">
+  <span
+    class="input-group-addon"
+    ng-disabled="datePicker.ngDisabled">
+      <i class="fa fa-calendar"></i>
+    </span>
+</div>
+<span
+  class="{{datePicker.clearBtnClass}} pointer"
```

Comment:
What's the purpose of the clearBtnClass? is it really needed?

Reply:
After re consideration, its not required, removed it.